dist/dropdown/ds4/dropdown.css

```diff
@@ -0,0 +1,225 @@
+.menu,
+.fake-menu,
+.combobox {
+  line-height: normal;
+  position: relative;
+}
+span.menu,
+span.fake-menu,
+span.combobox {
+  display: inline-block;
+}
+.menu button.expand-btn--borderless,
+.fake-menu button.expand-btn--borderless {
+  border-color: transparent;
+  vertical-align: initial;
+}
+.menu button.expand-btn--borderless[aria-expanded="true"] ~ .menu__items[role="menu"],
+.fake-menu button.expand-btn--borderless[aria-expanded="true"] ~ .menu__items[role="menu"] {
+  top: 41px;
+}
+.menu .expand-btn[aria-expanded="true"],
+.fake-menu .expand-btn[aria-expanded="true"],
+.combobox .expand-btn[aria-expanded="true"] {
+  border-bottom: 0;
+}
+.menu .expand-btn[aria-expanded="true"] ~ .menu__items[role="menu"],
+.combobox .combobox__control.combobox__control--expanded + .combobox__options[role="listbox"],
+.fake-menu .expand-btn[aria-expanded="true"] ~ .fake-menu__items {
+  display: block;
+}
+.menu .expand-btn ~ .menu__items[role="menu"],
+.combobox .combobox__control + .combobox__options[role="listbox"],
+.fake-menu .expand-btn ~ .fake-menu__items {
+  display: none;
+  position: absolute;
+  z-index: 1;
+}
+.menu__items[role="menu"],
+.combobox__options[role="listbox"],
+.fake-menu__items {
+  border: 1px solid;
+  box-shadow: 0 2px 4px 0 rgba(199, 199, 199, 0.5);
+  box-sizing: border-box;
+  min-width: calc(100%);
```

Comment:
is there a reason we need to use `calc()` in this .

Reply:
Surprisingly, yes, because it forces the browser to do a repaint/reflow based on actual contents _when they're shown_. Using just `min-width: 100%;` doesn't work correctly, for some reason.

But, feel free to test things out on your own. 👍